03-Calling-an-API/vue.config.js

```diff
@@ -0,0 +1,18 @@
+const webpack = require("webpack");
+
+module.exports = {
+  devServer: {
+    port: 3000,
+    proxy: "http://localhost:3001"
```

Comment:
what's this line for?

Reply:
This sample has an Express server running on port 3001; this line allows Webpack Dev Server to proxy calls to that URL when the URL doesn't match a route that it knows about.

So, the SPA, when calling the API, can just use `/api/external` as if it was a URL available on the same domain. This was one of the requirements that was discussed early-on when devising how the new samples should look.